var/spack/repos/builtin/packages/tau/package.py

```diff
@@ -124,6 +125,7 @@ class Tau(Package):
     depends_on("rocprofiler-dev", when="+rocprofiler")
     depends_on("roctracer-dev", when="+roctracer")
     depends_on("hsa-rocr-dev", when="+rocm")
+    depends_on("rocm-smi-lib", when="+rocm")
```

Comment:
Has this always been true (and just not noted properly) or is this new with this version?

Reply:
This only applies to `@2.32.1 +rocm` -- I updated the constraint just now and pushed.